ansible/roles/disable_hyperthreading/files/disableHT.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Disable HT at Runtime
+
+# see https://access.redhat.com/solutions/352663
+# preferred: /sys/devices/system/cpu/smt/control
+# fallback: /sys/devices/system/cpu/cpu
+
```

Comment:
Out of curiosity, in which case would we not have ``smt/control`` available ? Is it a matter of cloud provider, virtualization platform or OS version ?

Reply:
Replying to self, it seems like the only factor is the kernel version ? Looking at a limited sample of some nodes we have, I don't see smt available anywhere .. not even on my Fedora laptop ?

Reply:
Yes, kernel version.  I went ahead and handled older version just in case, and because my test cluster is on an old version and I didn't update.